plugin_store/database/database.py

```diff
@@ -67,6 +69,55 @@ def init(self):
         alembic_cfg = Config("/alembic.ini")
         command.upgrade(alembic_cfg, "head")
 
+    async def list_announcements(self, active: bool = True):
```

Comment:
Unusued var

Reply:
Yes, it's unused. i wanted to define the API of that function for future use to differentiate between 2 different endpoints that will use it. In future it'll add filters that will only list announcements that are currently live, that is enabled and/or in a specific timeframe.